README.adoc

```diff
@@ -0,0 +1,73 @@
+= About JICMP
+
+_JICMP_ is a small library to allow the use of _IPv4_ _ICMP_ (raw) packets in Java.
+
+== Build from source
+
+.Requirements
+
+* git
+* automake
+* autoconf
+* libtool
+* Java JDK 1.5+
+
+The repository has a _git_ submodule which contains _Macros_ required to compile from source code.
+
+.Clone the respository on local disk
+[source]
+----
+git clone https://github.com/OpenNMS/jicmp.git
+----
+
+.Switch into source code repository
+[source]
+----
+cd jicmp
+----
+
+.Initialize and update the git submodule.
+[source]
+----
+git submodule init
+git submodule update
+----
+
+.Update generated configuration files with
+[source]
+----
+autoreconf -fvi
+----
+
+.Generate make files and compile JICMP
+[source]
+----
+./configure
```

Comment:
We should install to `/usr` using `./configure--prefix=/usr` as some distros don't include `usr/local` to the library search path by default.